Test_Birdyboard.py

```diff
@@ -0,0 +1,17 @@
+import birdyboard
+
+test_that_there_is_a_main_menu(self):
+
+test_that_there_is_are_any_necessary_submenus(self):
+
+  test_that_a_Choose_User_submenu_appears_when_Select_User_is_entered_from_main_menu(self):
+
+  test_that_Confirmation_Menu_appears_when_Exit_is_entered_from_main_menu(self):
+
+    test_that_Confirmation_Menu_Choice_1_kills_birdyboard_program(self):
+
+    test_that_Confirmation_Menu_Choice_2_returns_user_to_main_menu(self):
```

Comment:
I'm going to recommend that you not start with testing the menu system. Not critical functionality.